src/thenCatchP.js

```diff
@@ -0,0 +1,28 @@
+import { invoker } from 'ramda';
+
+/**
+ * Composable shortcut for `Promise.then` that allows for success and failure callbacks.
+ * The thenCatchP function returns a Promise. It takes three arguments: a callback function for the success of the Promise,
+ * a callback function for the failure of the Promise, and the promise instance itself.
+ *
+ * @func thenCatchP
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.26.0|v2.26.0}
+ * @category Function
+ * @sig (a -> b) -> (c -> d) -> Promise a -> Promise b | d
+ * @param {Function} onFulfilled A Function called if the Promise is fulfilled. This function has one argument, the fulfillment value
+ * @param {Function} onRejected A Function called if the Promise is rejected. This function has one argument, the error
+ * @param {Promise} promise Any Promise or Thenable object
+ * @return {Promise} A Promise in the pending status
```

Comment:
I'd drop the description here. Promise doesn't need to be in pending status here. I can already be settled by fulfillment or rejection

```suggestion
 * @return {Promise}
```